schema/core.json

```diff
@@ -0,0 +1,65 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://docs.oasis-open.org/openeox/tbd/schema/core.json",
+  "title": "EOL and EOS Information",
+  "description": "A draft schema for representing End-of-Life (EOL) and End-of-Support (EOS) information in OpenEoX.",
+  "type": "object",
+  "$defs": {
+    "schema_t": {
+      "title": "OpenEoX Core schema",
+      "description": "Specifies the schema the JSON object must be valid against.",
+      "type": "string",
+      "enum": [
+        "https://docs.oasis-open.org/openeox/tbd/schema/core.json"
+      ]
+    },
+    "timestamp_t": {
+      "title": "Timestamp",
+      "description": "Contains the timestamp at which the product reaches the specified stage.",
+      "type": "string",
+      "oneOf": [
+        {
+          "format": "date-time"
+        },
+        {
+          "enum": [
+            "tba"
```

Comment:
This is still to be announced, is this something we need to revisit?

Reply:
This is the indicator that it is still to be announced. Resolve with cbaa1e8.